frontend/src/pages/AppDetails/AppDetailsPage.tsx

```diff
@@ -62,7 +62,7 @@ class AppDetails extends React.Component<AppDetailsProps, AppDetailsState> {
   constructor(props: AppDetailsProps) {
     super(props);
     const urlParams = new URLSearchParams(this.props.location.search);
-    const cluster = urlParams.get('cluster') || HomeClusterName;
+    const cluster = urlParams.get('cluster') || undefined;
```

Comment:
Shouldn't be the home cluster by default if the cluter query parameter is missing?.

Reply:
Originally the HomeClusterName in frontend was empty '' string and supposed to be removed,
we have it now by default in backend, when no cluster param is send, the backend takes the default conf.KubernetesConfig.ClusterName